hazelcast/src/main/java/com/hazelcast/sql/SqlResult.java

```diff
@@ -98,4 +98,12 @@ default boolean isRowSet() {
      */
     @Override
     void close();
+
+    /**
+     * Whether the result is possibly infinite, i.e. it may have an infinite number of rows, and creation of the next row
+     * may take infinite time. Set to {@code true} for Jet queries.
+     *
+     * @return {@code true} if the result is possibly infinite, {@code false} otherwise.
+     */
+    boolean isInfiniteRows();
```

Comment:
This is changing the public API. Is the SQL team OK with that? Can't we use the old internal API?